utils/prefix.ts

```diff
@@ -20,3 +20,6 @@ export function isRemark(prefix: string): boolean {
 
 export const hasMarketplace = (prefix: string): boolean =>
   ['rmrk', 'ksm', 'ahp', 'ahk'].includes(prefix)
+
+export const disabledOperationPrefixes = (prefix: string): boolean =>
```

Comment:
since it returns a boolean I would rather do`hasOperationsDisabled = (prefix: string): boolean`  or `hasPrefixOperationsDisabled`

Reply:
updated